portal/migrations/versions/68a25f790d27_.py

```diff
@@ -15,6 +16,8 @@
 
 
 def upgrade():
+    if current_app.config.get('GIL'):
```

Comment:
I think `PERSISTENCE_DIR` is the top-level config that sets `GIL`, but don't we need this to run on demo TNUSA, where `PERSISTENCE_DIR` is configured as `gil`?

Reply:
We happen to set both, verified in `https://demo.us.truenth.org/settings`.  I've just used this one in a number of like migrations.

Reply:
Sorry, misunderstood that was for the triggers table, not the authored datetime fixup